auth/src/main/java/org/aerogear/mobile/security/checks/EmulatorCheck.java

```diff
@@ -15,16 +15,19 @@
      * for x86' If the serial is equal to 'null' The manufacturer contains 'genymotion' If the brand
      * and device start with 'generic'
      *
-     * @return <code>true</code> if device is an emulator
+     * @return <code>true</code> if device is *not* running on an emulator and thus in a 'safer'
+     *         environment
      */
     @Override
     protected boolean execute(@NonNull Context context) {
-        return Build.FINGERPRINT.startsWith("generic") || Build.FINGERPRINT.startsWith("unknown")
+        final boolean isRunningOnEmulator = Build.FINGERPRINT.startsWith("generic")
+                        || Build.FINGERPRINT.startsWith("unknown")
                         || Build.MODEL.contains("google_sdk") || Build.MODEL.contains("Emulator")
                         || Build.MODEL.contains("Android SDK built for x86") || Build.SERIAL == null
                         || Build.MANUFACTURER.contains("Genymotion")
                         || (Build.BRAND.startsWith("generic") && Build.DEVICE.startsWith("generic"))
                         || "google_sdk".equals(Build.PRODUCT);
+        return !isRunningOnEmulator;
```

Comment:
no need to have the `isRunningOnEmulator` variable. We can just return.

Reply:
The extra name here is for self-documenting

Reply:
Good point 👍